source/multiple-mongosyncs.txt

```diff
@@ -133,6 +127,20 @@ To configure multiple ``mongosync`` instances:
            --cluster1 "mongodb://user:password@cluster1host:27500" \
            --id shard02 --port 27602
 
+      The connection strings for the :option:`--cluster0` and 
+      :option`--cluster1` options should point to ``mongos`` instances.
+      In the example, they use the same ``mongos`` instance.
+
+      Each ``mongosync`` instance:
+
+      - Connects to a ``mongos`` instance in the source cluster.
+      - Connects to a ``mongos`` instance in the destination cluster.
+      - Replicates a single shard from the source cluster, 
+        identified by the :option:`--id` option.
+      - Specifies a unique port to use for during synchronization.
```

Comment:
[extra word]

Looks like we can remove the "for" on line 140.

Reply:
Fixed